openedx/core/djangolib/tests/test_markup.py

```diff
@@ -157,3 +157,50 @@ def test_clean_dengers_html_filter(self):
         assert not html_soup.find('form')
         assert not html_soup.find('blink')
         assert not html_soup.find('object')
+
+
+class TestHTMLCleaner(unittest.TestCase):
+    """
+    Tests that Url links are being cleaned properly and no useful link is removed.
+    """
+
+    def setUp(self):
+        self.cleaner = HTMLCleaner(style=True, inline_style=False, safe_attrs_only=False)
+
+    def test_valid_urls(self):
```

Comment:
We can improve/simplify these tests by using[ Data Driven Tests](https://ddt.readthedocs.io/en/latest/). something like this
 ```suggestion
    @ddt.data(
        ('javascript:alert('hello') https://example.com', 'https://example.com'),
        ('http://example.com/path/to/page', 'http://example.com/path/to/page'),
    )
    @ddt.unpack
    def test_valid_urls(self, original_url, cleaned_url):
```